tests/spec/mesos_slave/mesos_slave_spec.rb

```diff
@@ -4,6 +4,30 @@
   it { should be_installed }
 end
 
-describe service('mesos-slave') do
-  it { should be_running }
+describe "mesos slave" do
+  describe service('mesos-slave') do
+    it { should be_running }
+  end
+
+  describe file('/etc/mesos/zk') do
+    it            { should be_file }
+    its(:content) { should contain ':2181/mesos' }
+  end
+
+  describe file('/etc/default/mesos-slave') do
+    it            { should be_file }
+    its(:content) { should match /^MASTER=`cat \/etc\/mesos\/zk`$/ }
+  end
+
+  describe file('/etc/mesos-slave') do
+    it { should be_directory }
+  end
+  describe file('/etc/mesos-slave/work_dir') do
+    it            { should be_file }
+    its(:content) { should contain '/tmp/mesos' }
```

Comment:
yeah will break if someone configures it differently but should be ok within the confines of a test harness